streamx-console/streamx-console-service/src/assembly/plugins/streamx.yml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: streamx
+  namespace: native-flink
```

Comment:
Maybe used default namespace is best to first deployed,this configuration let user to config it